frontend/packages/otr/src/pages/ClerkInterpreterOverviewPage.tsx

```diff
@@ -29,7 +29,7 @@ export const ClerkInterpreterOverviewPage = () => {
   const { overviewStatus, interpreter } = useAppSelector(
     clerkInterpreterOverviewSelector
   );
-  const { addStatus } = useAppSelector(qualificationSelector);
+  const { addStatus, removeStatus } = useAppSelector(qualificationSelector);
```

Comment:
`removeStatus` ei käytössä. Lisäksi kun teet rebasen niin github action ei taida mennä edes läpi jos näitä ylimäräisiä muuttujia määritelty. Näitä on joitakin lisää `clerkInterpreterOverview` sagassa

Reply:
Je, päivitetty